hummingbird/ml/_container.py

```diff
@@ -319,12 +330,65 @@ def score_samples(self, *inputs):
 
 
 # PyTorch containers.
-class PyTorchSklearnContainer(ABC):
+class PyTorchSklearnContainer(SklearnContainer):
     """
     Base container for PyTorch models.
-    We used this container to surface PyTorch-specific functionalities in the containers.
     """
 
+    def save(self, location):
+        assert self.model is not None, "Saving a None model is undefined."
+
+        if constants.TEST_INPUT in self._extra_config:
+            self._extra_config[constants.TEST_INPUT] = None
+
+        if "torch.jit" in str(type(self.model)):
+            assert not os.path.exists(location), "Directory {} already exists.".format(location)
+            os.makedirs(location)
+            self.model.save(os.path.join(location, "deploy_model.zip"))
+            model = self.model
+            self._model = None
```

Comment:
Yes this is necessary because otherwise dill fails. Basically I need to remove from the container all references to objects that cannot be dumped. No idea if there is a better way of doing this.